frontend/src/lib/util.ts

```diff
@@ -230,6 +235,59 @@ export function useErrorState(dependentSetter?: (...args: any) => void) {
   return [error, setError as any];
 }
 
+// Source Code: https://github.com/kubernetes/kubernetes/blob/2f275b72b2248bb7c522327115acb17223b8387b/staging/src/k8s.io/apimachinery/pkg/util/duration/duration.go#L48
+export function humanDuration(milliseconds: number): string {
```

Comment:
This will output very similar output to the `humanize-duration` package we've started using recently.
I don't see a need for this function since we already output a similar format + we allow for localized versions if needed.

Reply:
I like what you mentioned here. Let's use humanize-duration package. The purpose of this function is to display output the same as kubectl does. But some users told me they'd prefer to **output a similar format + we allow for localized versions**.